t/plugin/traffic-split2.t

```diff
@@ -376,28 +374,38 @@ location /t {
         local t = require("lib.test_admin").test
         local bodys = {}
         local headers = {}
+        local headers2 = {}
         headers["custom_header"] = "custom-one"
-        for i = 1, 4 do
+        headers2["custom_header"] = "customtow"
```

Comment:
Better to avoid using a word that looks like typo

Reply:
Updated.